src/Responses/FineTuning/RetrieveJobResponseHyperparameters.php

```diff
@@ -8,29 +8,35 @@
 use OpenAI\Responses\Concerns\ArrayAccessible;
 
 /**
- * @implements ResponseContract<array{n_epochs: int|string}>
+ * @implements ResponseContract<array{n_epochs: int|string, batch_size: int|string|null, learning_rate_multiplier: float|string|null, prompt_loss_weight: float|string|null}>
  */
 final class RetrieveJobResponseHyperparameters implements ResponseContract
 {
     /**
-     * @use ArrayAccessible<array{n_epochs: int|string}>
+     * @use ArrayAccessible<array{n_epochs: int|string, batch_size: int|string|null, learning_rate_multiplier: float|string|null, prompt_loss_weight: float|string|null}>
      */
     use ArrayAccessible;
 
     private function __construct(
         public readonly int|string $nEpochs,
+        public readonly int|string|null $batchSize,
+        public readonly float|string|null $learningRateMultiplier,
+        public readonly float|string|null $promptLossWeight,
```

Comment:
```suggestion

```

This parameter is only available on the deprecated `FineTune` endpoint and not on `FineTuning`.